### ISSUE TITLE
[css-paint-api] Try and find a way to access images w/o using a custom property

### ISSUE THREAD

**Issue by bfgeek** [w3c/css-houdini-drafts#24]
It might be desirable to have images loaded implicitly in a paint function, without having them loaded via. a custom property.

For example performing a network request, or a pull from the main execution context, to receive an image to use.

Thought: Are there any existing properties today which do this?

**Comment by tabatkins** [comment]
The only thing that's kinda close to this is that, theoretically, things like `list-style: bullet` can render the bullet with an image.  There's nothing else in CSS that creates something image-like without the user explicitly asking for a URL.

**Comment by FremyCompany** [comment]
Aren't bullets & co just unicode glyphs?

**Comment by bfgeek** [comment]
For us these are just paths painted onto the canvas.
https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/Source/core/paint/ListMarkerPainter.cpp&l=22

I can't think of anything which magically accesses a resource on the network magically.

**Comment by tabatkins** [comment]
Yeah, my point is that they're *allowed* to be that.  In actual browsers they're either text or manually-drawn stuff, but there's nothing preventing them from being a network-fetched url.  (It would just be a terrible idea.)

That's the only place where it's even possible, tho.  Nowhere else does a built-in CSS property even have a chance of fetching anything.

**Comment by bfgeek** [comment]
Moved to LVL 2
